### ISSUE TITLE
Lava Walker

### ISSUE THREAD

**Issue by soudblox** [Athlaeos/EnchantsSquaredRewritten#5]
I am on Minecraft 1.20.1 and I am using Paper.

I have applied the lava walker enchantment into my boots, but walking in lava does not work.

**Comment by Athlaeos** [comment]
Could you be a little more specific? What level did you test? Do you have other plugins installed? How did you test it?

**Comment by soudblox** [comment]
> Could you be a little more specific? What level did you test? Do you have other plugins installed? How did you test it?

I had Lava Walker II on my diamond boots.
Yes, I do have these plugins in my server.
![image](https://github.com/Athlaeos/EnchantsSquaredRewritten/assets/73058244/610b806d-1779-4768-984f-3a1c3ab0ef63)

I enchanted a book on the enchantment table and got the Lava Walker enchant. I combined my diamond boots with the Lava Walker enchant, tried to walk on lava, and nothing happened.
